bindata/network/infra-dpu-cni/000-ns.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-infra-dpu-cni
```

Comment:
I don't think we need a new namespace for this. Just put it in openshift-multus with the other misc CNI stuff